Gopkg.lock

```diff
@@ -16,8 +16,8 @@
 [[projects]]
   name = "github.com/aws/aws-sdk-go"
   packages = ["aws","aws/awserr","aws/awsutil","aws/client","aws/client/metadata","aws/corehandlers","aws/credentials","aws/credentials/ec2rolecreds","aws/credentials/endpointcreds","aws/credentials/stscreds","aws/defaults","aws/ec2metadata","aws/endpoints","aws/request","aws/session","aws/signer/v4","internal/shareddefaults","private/protocol","private/protocol/query","private/protocol/query/queryutil","private/protocol/rest","private/protocol/restxml","private/protocol/xml/xmlutil","service/s3","service/sts"]
-  revision = "25ef42b41b82230caae56ab23d872c81fb5c0eae"
-  version = "v1.12.46"
+  revision = "c13a879e75646fe750d21bcb05bf2cabea9791c1"
```

Comment:
Any reason why this file changed? Did you have to update or add an external library?